src/richie/apps/courses/templates/courses/cms/fragment_course_run.html

```diff
@@ -15,7 +15,7 @@
     {% if run|has_connected_lms %}
         <div
             class="richie-react richie-react--course-run-enrollment"
-            data-props='{"courseRunId": {{ run.id }}}'
+            data-props='{"courseRunId": {{ run.id }}, "loginUrl": "{% url 'social:begin' 'edx-oauth2' %}"}'
```

Comment:
I don't know python social auth, but [from what I read](https://python-social-auth.readthedocs.io/en/latest/use_cases.html#return-the-user-to-the-original-page), it requires a `next` url query param to return to this page after login.
Did I miss something ?

Reply:
No I just didn't start to search how to do it 😁

Reply:
Thank you math for the doc.
Related to #1105, I tried to but a next param in the url as explain in the doc and it works great!

`?next={{request.path}}`